### ISSUE TITLE
Core error

### ISSUE THREAD

**Issue by markfaine** [bacula-web/bacula-web#166]
**Describe the bug**

**To Reproduce**
1. Go to any page
2. See error

**Expected behavior**
Page to load normally

**Screenshots**
![image](https://user-images.githubusercontent.com/949949/226478733-0dbcfef6-72e3-4248-a88a-ceb31d48087c.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Tested on Firefox and Chrome
 - Version: Lastest docker container image

**Additional context**
Upgrade from a previous version

Config file:

$config[0]['label'] = 'Bacula Production';
$config[0]['host'] = 'my mysql host';
$config[0]['login'] = 'bacula';
$config[0]['password'] = 'my mysql password';
$config[0]['db_name'] = 'bacula';
$config[0]['db_type'] = 'mysql';
$config[0]['db_port'] = '3306';

# connection to mysql is fine from inside docker container:
```
# nc -zv my-mysql-host 3306
Connection to  my-mysql-host port [tcp/*] succeeded!
```

**Comment by dfranco** [comment]
Hi @markfaine ,

Thanks for the bug report.

Using latest 8.8.0 right ?
could you enable debug mode and share here the full error pls ?

running 

`docker-compose logs --tail 50`

could be helpful too, thanks for your feedback

**Comment by dfranco** [comment]
Hi @markfaine,

I'm able to reproduce the same issue by running Bacula-Web v8.8.0 on Ubuntu 22.04 with PHP 8.1.2.

Working on a fix, I'll keep you posted.

**Comment by dfranco** [comment]
@markfaine 
On my side, It appear that the "Core error" was due to missing users auth db setup, I'll make the error message more verbose.
But it could be due to other reasons for you, pls share the output once you've enabled debug mode.

**Comment by markfaine** [comment]
> 
Yes, I found the same solution.  The file was there but the permissions were wrong.  However, I'm still seeing a very brief error upon login:
![bacula_error](https://user-images.githubusercontent.com/949949/229907568-4131a22b-1caa-489d-950c-3f191fa1225b.png)
However, this doesn't affect loading of the application.

**Comment by dfranco** [comment]
Thanks a lot for your feedback @markfaine 

both issues will be fixed in separated issues before next release.

**Comment by dfranco** [comment]
Issue #168 is now fixed, #169 will be fixed in next release.

Closing this one
